module/enrichers.mjs

```diff
@@ -636,6 +637,95 @@ async function enrichSave(config, label, options) {
 
 /* -------------------------------------------- */
 
+/**
+ * Enrich an item use link to roll an item on the selected token.
+ * @param {string[]} config            Configuration data.
+ * @param {string} [label]             Optional label to replace default text.
+ * @returns {Promise<HTMLElement|null>}         An HTML link if the item link could be built, otherwise null.
+ *
+ * @example Use an item from a Name:
+ * ```[[/item Heavy Crossbow]]```
+ * becomes
+ * ```html
+ * <a class="roll-action" data-type="item">
+ *   <i class="fa-solid fa-dice-d20"></i> Heavy Crossbow
+ * </a>
+ * ```
+ *
+ * @example Use an Item from a UUID:
+ * ```[[/item Actor.M4eX4Mu5IHCr3TMf.Item.amUUCouL69OK1GZU]]```
+ * becomes
+ * ```html
+ * <a class="roll-action" data-type="item">
+ *   <i class="fa-solid fa-dice-d20"></i> Bite
+ * </a>
+ * ```
+ *
+ * @example Use an Item from a "Name UUID":
+ * ```[[/item Actor.Akra (Dragonborn Cleric).Item.Mace]]```
+ * becomes
+ * ```html
+ * <a class="roll-action" data-type="item">
+ *   <i class="fa-solid fa-dice-d20"></i> Mace
+ * </a>
+ * ```
+ *
+ * @example Use an Item from a Relative UUID:
+ * ```[[/item .amUUCouL69OK1GZU]]```
+ * becomes
+ * ```html
+ * <a class="roll-action" data-type="item">
+ *   <i class="fa-solid fa-dice-d20"></i> Bite
+ * </a>
+ * ```
+ */
+async function enrichItem(config, label) {
+  const givenItem = config.values.join(" ");
+  // If config is a UUID
+  const itemUuidMatch = givenItem.match(/^(?<synthid>Scene\.\w{16}\.Token\.\w{16}\.)?(?<actorid>Actor\.\w{16})(?<itemid>\.?Item(?<relativeId>\.\w{16}))$/);
+  if (itemUuidMatch) {
+    const ownerActor = itemUuidMatch.groups.actorid.trim();
+    if (!label) {
+      if ((await fromUuid(givenItem)) == null) {
+        console.warn(`Item not found while enriching ${givenItem}.`);
+        return null;
+      } else label = (await fromUuid(givenItem)).name;
```

Comment:
We shouldn't be calling `fromUuid` here twice. There is also no need for `else` when we always return within the `if` statement.

```suggestion
      const item = await fromUuid(givenItem);
      if ( !item ) {
        console.warn(`Item not found while enriching ${givenItem}.`);
        return null;
      }
      label = item.name;
```